router/voteCallbacks.go

```diff
@@ -11,8 +11,10 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
-const MaxListSize = 256
-const MaxListIterations = 64
+const (
+	MaxListSize       = 256
+	MaxListIterations = 64
```

Comment:
Remove `MaxListIterations` since we don't need it anymore (AFAIK)

Reply:
It's useful in the CLI as a paging loop limit, but I can move it there, maybe makes more sense.

Reply:
But there is something I don't get... Why MaxListSize is 256? The maximum we were using for pagination was 64, not 256. The point behind this commit is to get rid of MaxListIterations since it is not used anywhere in the code, right? Can you clarify please?

Reply:
Well, if the client want more, they can just the pagination `from` field on the next call.

I think 64 is a good number since printing more than 64 proceses on the client side looks a bit too much for a friendly UX.

Reply:
Ok, I'm doing a new  PR to decrease it then...